author	Joohi Rastogi <joohi.rastogi@stericsson.com>
commit	2d392db690639a9304a33aa6b32808ec2e8bce7d
	Thu, 5 Apr 2012 09:52:18 +0000 (12:52 +0300)

AVRCP: Fix response length for Unit Info command

This affect PTS Test Case TC_TG_ICC_BV_01_I

author	Joohi Rastogi <joohi.rastogi@stericsson.com>
commit	91baac990d2544348c2e4b221c79af5708cfcb5d
	Thu, 5 Apr 2012 09:52:27 +0000 (12:52 +0300)

AVRCP: Fix response length for Subunit Info command

This affect PTS Test Case TC_TG_ICC_BV_02_I

diff -cr bluez-4.99.pan.orig/audio/avctp.c bluez-4.99.pan/audio/avctp.c
*** bluez-4.99.pan.orig/audio/avctp.c	Mon Oct 15 03:24:39 2012
--- bluez-4.99.pan/audio/avctp.c	Mon Oct 15 03:26:22 2012
***************
*** 270,276 ****
  
  	DBG("reply to AVC_OP_UNITINFO");
  
! 	return 0;
  }
  
  static size_t handle_subunit_info(struct avctp *session,
--- 270,276 ----
  
  	DBG("reply to AVC_OP_UNITINFO");
  
! 	return operand_count;
  }
  
  static size_t handle_subunit_info(struct avctp *session,
***************
*** 294,300 ****
  
  	DBG("reply to AVC_OP_SUBUNITINFO");
  
! 	return 0;
  }
  
  static struct avctp_pdu_handler *find_handler(GSList *list, uint8_t opcode)
--- 294,300 ----
  
  	DBG("reply to AVC_OP_SUBUNITINFO");
  
! 	return operand_count;
  }
  
  static struct avctp_pdu_handler *find_handler(GSList *list, uint8_t opcode)
